scapy/layers/inet6.py

```diff
@@ -57,9 +57,11 @@
     StrLenField,
     X3BytesField,
     XBitField,
+    XByteField,
     XIntField,
     XShortField,
 )
+
```

Comment:
Hi,
I know it looks like nothing but we try to keep patches as tiny as possible (to keep our history as clean as possible and avoid unneeded issues for people maintaining draft PRs), and remove unnecessary changes
```suggestion
```

Reply:
Can I find formatting guidlines somewhere? If I run a formatter on the file, the whole file changes.